Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -287,20 +288,26 @@ void OnRecvPropertyRequest()
         {
         }
 
-        void SceneUpdated()
-        {
-            onSceneUpdateEnd -= SceneUpdated;
-
+#if UNITY_EDITOR
+        internal override void AfterUpdateScene() {
```

Comment:
Was the event implementation causing issues? I find it a bit strange to override a more general purpose virtual method in a properties partial class.

Reply:
I could use the `onSceneUpdateEnd` event but considering this is a derived class it seems better to use inheritance and leave events for external dependencies, no?

Reply:
I am not entirely sure that converting `SceneUpdated` here to an override `AfterUpdateScene` will produce exactly the same behavior.

Perhaps we can split this PR into three ?
1. C## and cpp code changes related to the new feature (user script callback) 
2. The refactoring around `SceneUpdated`
3. Changes related to `IgnoreNextNewSession` or (`ForceDeleteChildrenInNextSession`).

Reply:
I've created some PRs to break stuff out if you want to merge those first:
https://github.com/unity3d-jp/MeshSync/pull/841
https://github.com/unity3d-jp/MeshSync/pull/840/files
https://github.com/unity3d-jp/MeshSync/pull/842/files

Reply:
Thank you. The above PRs have been merged.